site/content/en/docs/Tasks/addons.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Addons"
+date: 2019-07-31
+weight: 4
+description: >
+  Using addons
+---
+
+minikube has a set of built in addons that can be used enabled within Kubernetes. 
```

Comment:
minikube has a set of built-in addons that, when enabled, can be used within Kubernetes